pkg/proxy/ipvs/proxier.go

```diff
@@ -1862,6 +1852,12 @@ func (proxier *Proxier) createAndLinkeKubeChain() {
 	existingNATChains := proxier.getExistingChains(proxier.iptablesData, utiliptables.TableNAT)
 
 	// Make sure we keep stats for the top-level chains
```

Comment:
This comment still belongs below I think.

Reply:
I think KubeMarkDropChain and KubeMarkMasqChain also chains that we should keep stat?